FirebaseAdmin/FirebaseAdmin/Auth/FirebaseUserManager.cs

```diff
@@ -62,14 +62,19 @@ public static FirebaseUserManager Create(FirebaseApp app)
         }
 
         /// <summary>
-        /// Get a user by id.
+        /// Gets the user data corresponding to the given user ID.
         /// </summary>
-        /// <param name="uid">The query user id.</param>
+        /// <param name="uid">A user ID string.</param>
         /// <param name="cancellationToken">For asynchronous operation.</param>
         /// <returns>A record of user with the queried id if one exists.</returns>
         public async Task<UserRecord> GetUserById(
             string uid, CancellationToken cancellationToken = default(CancellationToken))
         {
+            if (string.IsNullOrEmpty(uid))
+            {
+                throw new ArgumentException("Failed to get user. User id cannot be null or empty");
+            }
+
             const string getUserPath = "accounts:lookup";
             var payload = new Dictionary<string, object>()
```

Comment:
I believe this service actually responds with an array of user record objects. So we need to check if that array is empty, and if not return the first entry. But you can do that in a future PR if you want to.

Reply:
Yes, the java implementation has a response.getUsers() implementation. I thought since each user id is unique it would return a single user record or null but I can change that in a future PR